runtime/src/snapshot_package.rs

```diff
@@ -93,6 +93,24 @@ impl AccountsPackage {
         )
     }
 
+    /// Package up fields needed to verify an accounts hash
+    #[must_use]
+    pub fn new_for_accounts_hash_verifier(
+        package_type: AccountsPackageType,
```

Comment:
why have this as an argument? I thnk if it should always be `AccountsHashVerifier` variant, just set use that explicitly inside.